### DISCUSSION TITLE
Running inngest with https dev server?

### DISCUSSION THREAD

**Discussion by francistogram** [inngest/inngest#440] [Q&A] [answered]
Because a bunch of social media APIs only work with https:// I've setup my NextJS server to be https://assembly.local:3000/

I followed the directions here: https://www.inngest.com/docs/local-development#inngest-dev-server

Using `npx inngest-cli@latest dev -u https://assembly.local:3000/api/inngest`

I've verified that the endpoint actually does work using Postman
![image](https://user-images.githubusercontent.com/3721449/236688247-b3a8bdb4-296a-47c4-8f04-fb920bb547e7.png)

I'm unable to get inngest to connect via the commandline
![image](https://user-images.githubusercontent.com/3721449/236688252-50f26b4d-cc51-497b-bcae-0a8497946af6.png)

I've also started my server on the normal `http://localhost:3000/` and everything works fine which is what leads me to think that it's likely an https issue for dev

Any idea what the issue might be here?

**Comment by yasyf** [comment, marked as answer]
I think this is unrelated to inngest; you'll need to set up local SSL termination! the easiest way is probably to use `ngrok`, point your social media APIs to the SSL endpoint, let it handle the termination, and then just run the local devsever as normal without SSL.

**Comment by francistogram** [comment]
Sounds good, went ahead and just changed everything to use `ngrok` and things work fine

Thanks!
